src/script/components/MessagesList/Message/ContentMessage/MessageActions/MessageReactions/MessageReactionsList.tsx

```diff
@@ -50,7 +53,8 @@ const MessageReactionsList: FC<MessageReactionsListProps> = ({reactions, ...prop
 
         return (
           <EmojiPill
-            emojiCount={users.length}
+            qualifiedIds={users}
+            conversationUsers={conversationUsers}
```

Comment:
Since we're passing all the conversation users anyway, would that make sense to filter out those that have not reacted at this point? And instead of `qualifiedIds` pass `reactionUsers` list of user entitties? 🤔